routes/conversations.js

```diff
@@ -52,8 +54,35 @@ exports.insertConversation = function(req, res) {
         .catch(error => res.status(error.code || 500).json(error));
 };
 
-exports.updateConversation = function(req, res) {
+exports.updateConversation = async function(req, res) {
```

Comment:
You could probably make codeclimate happy by extracting this new code as a separate function